Dependency of Storage to Model is not shown in DG #100

ernestlim8 opened the pull request:
Fixes #100

damithc reviewed (changes requested):
Thanks for the PR @ernestlim8 
Shouldn't there be a png file too?

Commit message: a full stop is missing.

ernestlim8 committed:
Dependency of Storage to Model is not shown in DG

Storage component depends on the model component because
JsonAddressBookStorage class uses the ReadOnlyAddressBook class.

Let's add the dependency in the architecture diagram to reflect this
dependency in the architecture diagram.

damithc commented:
Sorry for getting back so late @ernestlim8 
This is actually a weaker link than the other links. Perhaps a dashed arrow is more appropriate? Similar to how we show dependencies in class diagrams?

damithc reviewed (approved):
LGTM

damithc commented:
Thanks for the PR @ernestlim8 👍
